lib/hocon/impl/config_delayed_merge_object.rb

```diff
@@ -170,4 +170,8 @@ def hash
     # note that "origin" is deliberately NOT part of equality
     @stack.hash
   end
+
+  def render_to_sb(sb, indent, at_root, at_key, options)
```

Comment:
I get very confused when I look at this, but I think where I came from was I looked at `protected void render(StringBuilder sb, int indent, boolean atRoot, String atKey, ConfigRenderOptions options)` in `ConfigDelayedMerge`, saw the signature,  and found the corresponding ruby method `render_to_sb(sb, indent, at_root, at_key, options)`